src/app/cli/src/graphql.ml

```diff
@@ -215,11 +277,11 @@ module Make (Commands : Coda_commands.Intf) = struct
           ~fields:(fun _ ->
             [ pubkey_field ~resolve:(fun _ account ->
                   (* Hack: Account.Poly.t is only parameterized over 'pk once
-                   * and so, in order for delegate to be optional, we must also
-                   * make account public_key optional even though it's always
-                   * Some. In an attempt to avoid a large refactoring, and also
-                   * avoid making a new record, we'll deal with a value_exn here
-                   * and be sad. *)
+                             * and so, in order for delegate to be optional, we must also
```

Comment:
extra tabs here

Reply:
I think I saw this in your PR too

Reply:
maybe this is the formatter? seems fine then